examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3067,6 +3067,19 @@ server cluster ColorControl = 768 {
   command StepColorTemperature(StepColorTemperatureRequest): DefaultSuccess = 76;
 }
 
+server cluster BallastConfiguration = 769 {
+  readonly attribute int8u physicalMinLevel = 0;
+  readonly attribute int8u physicalMaxLevel = 1;
+  attribute int8u minLevel = 16;
+  attribute int8u maxLevel = 17;
+  readonly attribute int8u lampQuantity = 32;
```

Comment:
Looks like this is not enabling any of the optional attributes, right?  Should probably enable those for all-clusters-app, and enable this cluster with only mandatory attributes for all-clusters-minima-app.

Reply:
done